pages/about.html

```diff
@@ -4,9 +4,42 @@
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
+  <title>About</title>
+  <link href="/styles/style.css" rel="stylesheet">
+  <script src="https://kit.fontawesome.com/3e8fd03e4e.js"
+  crossorigin="anonymous"></script>
 </head>
 <body>
-  
+<header>
+<h1>Michelle's Portfolio Website</h1>
+  <nav>
+    <ul class="top-bottom-links">
+      <li>About</li>
+      <li><a href="/pages/index.html">Main</a></li>
+      <li><a href="/pages/portfolio.html">Portfolio</a></li>
```

Comment:
It's a bit distracting to me that the order of the nav links changes from page to page. Generally, we want to let users use their positional memory when navigating our site (or using an application), and having elements that rearrange themselves can detract from this.